ARM: dts: sunxi: h2-plus-bananapi-m2-zero: Power led heartbeat
    
Make the BananaPi M2 Zero's powerled blink so that we can see whether the kernel is running.

diff --git a/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts b/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts
index 6cb1cab0c..942fde97b 100644
--- a/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts
+++ b/arch/arm/boot/dts/sun8i-h2-plus-bananapi-m2-zero.dts
@@ -42,8 +42,8 @@
 
 		pwr_led {
 			label = "bananapi-m2-zero:red:pwr";
-			gpios = <&r_pio 0 10 GPIO_ACTIVE_LOW>; /* PL10 */
-			default-state = "on";
+			gpios = <&r_pio 0 10 GPIO_ACTIVE_HIGH>; /* PL10 */
+			linux,default-trigger = "heartbeat";
 		};
 	};
 
